README.md

```diff
@@ -1,59 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> One paragraph statement about the project.
+> This project is about leraning how to set up linters and how to use gitflow.
```

Comment:
- [ ] You have done a great job with your README but I can see a few typos and this could be seen as unprofessional. Remember that this is the first thing recruiters and team members will see when they check your repository. Please, fix the typos to make your README look even better :smiley: